.github/dependency-review.yml

```diff
@@ -10,4 +10,23 @@ fail-on-severity: moderate
 fail-on-scopes: runtime
 
 # Here is the list of licenses we want to deny and not use in our software. 
-deny-licenses: GPL-1.0-only,GPL-1.0-or-later,GPL-2.0-only,GPL-2.0-or-later,GPL-3.0-only,GPL-3.0-or-later,AGPL-1.0-only,AGPL-1.0-or-later,AGPL-3.0-only,AGPL-3.0-or-later,LGPL-2.0-only,LGPL-2.0-or-later,LGPL-2.1-only,LGPL-2.1-or-later,LGPL-3.0-only,LGPL-3.0-or-later
+deny-licenses:
```

Comment:
We might consider expanding this to spdx equivalent 

```
RPL-1.1
RPL-1.5
RPSL-1.0
OCLC-2.0
NGPL
ODbL-1.0
MS-RL
OSL-1.0
OSL-1.1
OSL-2.0
OSL-2.1
OSL-3.0
MPL-2.0
MPL-1.0
MPL-1.1
MPL-2.0-no-copyleft-exception
EPL-1.0
EPL-2.0
```

I rather like the allow list approach: https://github.com/ctcampbell/blueoak-to-dependency-review/blob/main/dist/blueoak-licenses.yml

Reply:
Completely valid request and I have set it to use that list now